source/includes/steps-fle-convert-to-a-remote-master-key.yaml

```diff
@@ -176,7 +176,6 @@ content: |
 title: Update the JSON Schema
 ref: update-the-json-schema
 content: |
-  The development JSON Schema directly referenced the development data
-  key id, so you need to generate a new JSON Schema. To do so, repeat
-  the :ref:`Define a JSON Schema <fle-define-a-json-schema>` step with
-  the new data key that was created with the remote master key.
+  If you have embedded the base64 key id for your data encryption key, you will
+  need to update your :ref:`JSON Schema <fle-define-a-json-schema>` with the new
+  base64 key id of your data encryption key.
```

Comment:
Have we defined the term "embedded" in this doc or step file? Perhaps it would be useful to specify the two different ways they may be referencing the data key -- either by query, or hardcoded schema. Perhaps something like this:

"Replace all references to the old data encryption key in your JSON schema with the new one you generated in the previous step whether directly embedded in your code or programmatically read."